setup.py

```diff
@@ -37,6 +38,7 @@
         "pytest",
         "pytest-cov",
         "pytest-mock",
+        "responses",
```

Comment:
Did you end up using this in tests? (also present in `tox.ini`)

Reply:
Ha, that's funny. It was used in tests for functions that got deleted in #56. Now it's not used any more. I removed it in 
2fa3f95